tests/travis/test.yaml

```diff
@@ -9,7 +9,7 @@
   skip: ["image.py", "image_rgba.py", "ajax_source.py"]
 - path: "plotting/server"
   type: server
-  skip: ["image.py", "image_rgba.py", "remote_image.py", "serversource.py", "server_source_upload.py", "embedding_example.py", "abstractrender.py", "census.py"]
```

Comment:
They should be excluded from tests (because I don't want to deal with trying to set up a blaze server on our testing right now) but I don't think they should be removed, because they aren't *broken*, the just need a separate server.